tests/unit/test.program.js

```diff
@@ -73,8 +73,8 @@ describe('program.Program', () => {
       shouldExitProgram: true,
     })
       .then(() => {
-        assert.equal(fakeProcess.exit.called, true);
-        assert.equal(fakeProcess.exit.firstCall.args[0], 1);
+        sinon.assert.calledOnce(fakeProcess.exit);
+        sinon.assert.calledWithMatch(fakeProcess.exit, 1);
```

Comment:
The sinon.js documentation shows sinon.match(number) requires the value to be "==" to the given number. In this case, it seems like we are checking for a specific number value.

calledWith() passes and throws a similar error message as calledWithMatch(), but I am not sure that it is performing the "==" check. 

To put it another way…here are the tests and errors when the values do not match.

1.Test -  sinon.assert.calledWith(fakeProcess.exit, 0);
1.Error - "AssertError: expected exit to be called with arguments 1 0"

2.Test - sinon.assert.calledWith(fakeProcess.exit, sinon.match(0));
2.Error - "AssertError: expected exit to be called with arguments 1 match(0)"

3.Test -  sinon.assert.calledWithMatch(fakeProcess.exit, 0);
3.Error - "AssertError: expected exit to be called with match 1 0”

I know that #2 and #3 are identical, other than that #3’s error message reads better. After digging into this a bit more. I am unsure of what sets #1 apart from the other two. I chose to use calledWithMatch() to make sure I was checking for a specific value, not just a number.